Abot2/Core/HyperLinkParser.cs

```diff
@@ -44,8 +44,16 @@ public virtual IEnumerable<HyperLink> GetLinks(CrawledPage crawledPage)
 
             var timer = Stopwatch.StartNew();
 
-            var links = GetHyperLinks(crawledPage, GetRawHyperLinks(crawledPage));
+            var links = new List<HyperLink>();
 
+            if (Config.IsFollowMetaRedirectsEnabled)
```

Comment:
Feels like none of this would be necessary if we use the crawledPage.RedirectedTo property instead